RecoMuon/MuonIdentification/src/MuonShowerDigiFiller.cc

```diff
@@ -0,0 +1,153 @@
+//
+// Package:    MuonShowerDigiFiller
+// Class:      MuonShowerDigiFiller
+// 
+/**\class MuonShowerDigiFiller MuonShowerDigiFiller.cc RecoMuon/MuonIdentification/src/MuonShowerDigiFiller.cc
+
+ Description: Class filling shower information using DT and CSC digis 
+
+ Implementation:
+     <Notes on implementation>
+*/
+//
+// Original Author:  Carlo Battilana, INFN BO 
+//         Created:  Sat Mar 23 14:36:22 CET 2019
+//
+//
+
+
+// system include files
+
+// user include files
+#include "RecoMuon/MuonIdentification/interface/MuonShowerDigiFiller.h"
+
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+
+//
+// constructors and destructor
+//
+
+MuonShowerDigiFiller::MuonShowerDigiFiller(const edm::ParameterSet& iConfig, edm::ConsumesCollector&& iC) : 
+  m_digiMaxDistanceX(iConfig.getParameter<double>("digiMaxDistanceX"))
+
+{
+
+  edm::InputTag label = iConfig.getParameter<edm::InputTag>("dtDigiCollectionLabel");
```

Comment:
```Label``` could be omitted and this line can also be joined with the next without a named temporary ```label`` variable.

Reply:
OK